README.md

```diff
@@ -8,6 +8,12 @@ We conduct ongoing audits that review all changes to the code to ensure a high d
 
 |  Date   | Audit                                                                                 |                                                       Mitigations                                                       | Scope                                                                 |
 | :-----: | :------------------------------------------------------------------------------------ | :---------------------------------------------------------------------------------------------------------------------: | :-------------------------------------------------------------------- |
+| 2024-07 | [Ackee Blockchain](audits/2024-07%20Ackee%20Blockchain.pdf)                           | [Here](https://bright-ambert-2bd.notion.site/6500bd49bd9a4e139936e8ebb97e592d?v=0d59d1bd0de145fa9e731c0d179219db&pvs=4) | `axelar-gmp-sdk-solidity` `AxelarAmplifierGateway`
```

Comment:
```suggestion
| 2024-07 | [Ackee Blockchain](audits/2024-07%20Ackee%20Blockchain.pdf)                           | [Here](https://bright-ambert-2bd.notion.site/6500bd49bd9a4e139936e8ebb97e592d?v=0d59d1bd0de145fa9e731c0d179219db&pvs=4) | `AxelarAmplifierGateway`
```
we can skip the repo, since we aren't adding it below